pull_request_template.md

```diff
@@ -1,5 +1,7 @@
 ## Description
 Completes User Story #
+Test Percentage:
+Number of Test:
```

Comment:
This is already part of the template: at the bottom, in the code block, you can just copy and past your RSpec results